pangram/pangram.py

```diff
@@ -1,2 +1,9 @@
 def is_pangram(sentence):
-    pass
+    str(sentence)
```

Comment:
I think this line can be dropped. No object is created and sentence is already a string.